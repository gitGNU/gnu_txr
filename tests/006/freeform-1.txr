@(collect)
@first @second @third
@(freeform)
@(coll)@{val /[0-9]+/}@(until)@\n@\n@(end)
@(end)
