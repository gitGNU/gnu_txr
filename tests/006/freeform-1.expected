first[0]="a"
first[1]="d"
first[2]="g"
second[0]="b"
second[1]="e"
second[2]="h"
third[0]="c"
third[1]="f"
third[2]="i"
val_0[0]="1"
val_1[0]="2"
val_0[1]="3"
val_1[1]="4"
val_0[2]="5"
val_1[2]="6"
val_2[2]="7"
val_3[2]="8"
